schedule/yast/yast_no_self_update/yast_no_self_update@s390x_kvm.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: yast_no_self_update
+description:    >
+  Test suite conducts installation with self-update explicitly disabled. No
+  hard checks are done that the self-updating is really disabled.
+  Installation is validated by successful boot and that YaST does not report
+  any issue.
+vars:
+  INSTALLER_NO_SELF_UPDATE: 1
+  YUI_REST_API: 1
+schedule:
+  product_selection:
+    - installation/product_selection/install_SLES
+    - installation/validate_no_self_update
+  installation:
+    - installation/installation_settings/validate_default_target
```

Comment:
I sense your motivation for rewriting the whole installation block, seems you want to recreate the "original" schedule and in the default file there is no hook to apply `validate_default_target` at this exact position.
On the other hand, if I look at all those tests between `validate_ssh_service_enabled` and `launch installation` they all start on the "Installation settings" page and return to this screen, so there won't be any loss of test coverage if you use the `default_systemd_target` hook from the default file and then don't overwrite the `installation` section.

Reply:
I've modifiled it, is that ok?